pages/topics/client-distribution.mdx

```diff
@@ -0,0 +1,398 @@
+# Client Distribution
+
+While mobile clients can be distributed through most system app stores, desktop clients require a custom solution for downloading and updating. Discord provides various APIs for downloading clients, their native modules, and keeping everything up-to-date. Visit the [downloads page](https://discord.com/download) to learn more about the clients currently offered.
+
+###### Distribution Base URLs
+
+These URLs, provided for convenience, provide access to all client downloads (and redirect access to [CDN-hosted ones](/reference#cdn-formatting)), but are NOT used for the API requests below. For those, see the [API Base URL](/reference#base-url).
+
+```
+https://dl.discordapp.net/
+https://dl-ptb.discordapp.net/
+https://dl-canary.discordapp.net/
+https://dl-development.discordapp.net/
+```
+
+## Clients
+
+### Web
+
+Discord offers a web client that can be used in a browser. This same web client is also used in the desktop client.
```

Comment:
> This same web client is also used in the desktop client.
Should we clarify here that the desktop client uses native modules for some things

Reply:
Done!